src/Model.php

```diff
@@ -776,33 +776,64 @@ public function setMulti(array $fields)
      *
      * @return mixed
      */
-    public function get(string $field = null)
+    public function get(string $fieldName = null)
     {
-        if ($field === null) {
+        if ($fieldName === null) {
             // Collect list of eligible fields
             $data = [];
             if ($this->only_fields) {
                 // collect data for actual fields
-                foreach ($this->only_fields as $field) {
-                    $data[$field] = $this->get($field);
+                foreach ($this->only_fields as $fieldName) {
+                    $data[$fieldName] = $this->get($fieldName);
                 }
             } else {
                 // get all fields
-                foreach ($this->getFields() as $field => $f) {
-                    $data[$field] = $this->get($field);
+                foreach ($this->getFields() as $fieldName => $field) {
+                    $data[$fieldName] = $this->get($fieldName);
                 }
             }
 
             return $data;
         }
 
-        $this->checkOnlyFieldsField($field);
+        // if field is not existing maybe it is a traversing link or aggregate function
+        if (!$this->hasField($fieldName)) {
+            // alias for count aggregate, consistent with Model scopes
+            if ($fieldName === '#') {
+                $fieldName = 'count(*)';
```

Comment:
can we drop `#` then completely?

Reply:
We can do anything we want 😄 
But it is good to have one special short symbol for frequent use:
`$country->action('count')->getOne();`
can be written now
`$country->get('#');`

Reply:
you have now introduced `$country->get('count(*)');`

Reply:
Yep